config/deploy.rb

```diff
@@ -0,0 +1,45 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.11.0"
+
+set :application, "asana"
+set :repo_url, "https://github.com/ITAR089IF/project-management-tool"
+
+# Default branch is :master
+ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+
+# Default deploy_to directory is /var/www/my_app_name
+set :deploy_to, "/var/www/deploy"
```

Comment:
how's it going with permissions?

Reply:
all is ok

Reply:
hm.. interesting.. r u deploying as root?

Reply:
I fix troubles with user

Reply:
@BogdanBusko if you in some way do "fix" any issues - they must be listed in the README, otherwise - what's the purpose if nobody but you will be able to deploy?